uint32 serial_number #serial number of the board to control
int32 breaker_number ## 0=Base, 1=RightArm 2=LeftArm
string command  # Options start, stop, reset, disable, none
uint32 flags
---
int32 retval # 0 = false, 1=true
